python/mxnet/ndarray_doc.py

```diff
@@ -54,6 +54,268 @@ class ReshapeDoc(NDArrayDoc):
     (4L, 3L, 2L)
     """
 
+class ConcatDoc(NDArrayDoc):
+    """
+    Examples
+    --------
+    Joins input arrays along a given axis.
+    >>> x = mx.nd.array([[1,1],[2,2]])
```

Comment:
I think you need an extra newline here before starting the example. Check http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-14243/4/api/python/ndarray/ndarray.html#mxnet.ndarray.Concat

Reply:
I meant before `<<<`. 
You can check the preview here: http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-14243/6/api/python/ndarray/ndarray.html#mxnet.ndarray.Concat